CHANGELOG.md

```diff
@@ -19,15 +19,17 @@ Another example [here](https://co-pilot.dev/changelog)
 - Add CHANGELOG.md ([#84](https://github.com/chingu-x/chingu-dashboard-be/pull/84))
 - Add Role/Permission guard ([#97](https://github.com/chingu-x/chingu-dashboard-be/pull/97))
 - Add e2e tests for auth controller ([#102](https://github.com/chingu-x/chingu-dashboard-be/pull/102))
+- Add e2e tests for techs controller ([#103](https://github.com/chingu-x/chingu-dashboard-be/pull/103))
```

Comment:
change #103 is duplicated
could you put your change in the right order please, i.e. most recent change in the bottom